db/etl/import_us_cities_advanced.py

```
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+"""
+Advanced script to import US cities from the countries-states-cities-database
+using TimezoneFinder for precise timezone determination based on coordinates.
+Uses psycopg3 for database connectivity.
+"""
+
+import os
+import requests
+import psycopg
+from psycopg.rows import dict_row
```

Ruff (0.8.2)

`psycopg.rows.dict_row` imported but unused

Remove unused import: `psycopg.rows.dict_row`

(F401)